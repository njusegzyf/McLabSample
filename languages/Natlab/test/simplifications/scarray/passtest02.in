
if A | B
    foo();
end