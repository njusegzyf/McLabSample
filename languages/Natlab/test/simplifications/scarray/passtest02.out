
t1 = A;
if t1
    t3 = t1;
else
    t2 = B;
    t3 = t1 | t2;
end
if t3
    foo();
end
